modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1214,6 +1214,53 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-156704 - Verify that the user cannot insert an invalid date in an object field."
+	@priority = "5"
```

Comment:
Here stays with `@priority = "2"`. Could you change it again?

```
@description = "LPS-156704 - Verify that the user cannot insert an invalid date in an object field."
@priority = "2"
test CannotInsertInvalidDate {

```

I'll describe it better in the next review. Thanks for your patience. 😄 
cc: @kristinonias

Reply:
No worries! Changed back to priority 2.